python/ray/services.py

```diff
@@ -165,6 +165,31 @@ def all_processes_alive(exclude=[]):
     return True
 
 
+def hostname_to_ip_address(address):
+    """Convert a hostname to a numerical IP addresses in an address.
```

Comment:
I would update the comment to say that the address format is basically ip or hostname followed by an optional `:port`. I.e. expected input "{FQDN | IP_ADDR}[:PORT]?"

Reply:
I document this in the args section. I didn't put it in the top string because of space constraints.